pallets/pallet-bonded-coins/src/tests/transactions/mint_into.rs

```diff
@@ -1 +1,705 @@
+use core::u128;
 
+use frame_support::{
+	assert_err, assert_ok,
+	traits::{
+		fungibles::Inspect,
+		tokens::{Fortitude, Preservation},
+	},
+};
+use frame_system::{pallet_prelude::OriginFor, RawOrigin};
+use sp_runtime::{assert_eq_error_rate, ArithmeticError, Permill, TokenError};
+
+use crate::{
+	curves::{polynomial::PolynomialParameters, Curve},
+	mock::{runtime::*, *},
+	types::{Locks, PoolStatus},
+	Error,
+};
+
+const MAX_ERROR: Permill = Permill::from_perthousand(1);
+
+#[test]
+fn mint_first_coin() {
+	let pool_id = calculate_pool_id(&[DEFAULT_BONDED_CURRENCY_ID]);
+
+	let curve = get_linear_bonding_curve();
+
+	let initial_collateral = 100u128;
+	let amount_to_mint = 1u128;
+	let expected_price = collateral_at_supply(amount_to_mint);
```

Comment:
Good question, but at least in the tests it may make sense to check that we are not toooooo far off? Just to see that it performs close to expectations?